Eshop.WebApi/Controllers/CategoryController.cs

```diff
@@ -16,38 +24,46 @@ public class CategoryController : Controller
         [HttpGet]
         public List<Category> GetCategories()
         {
-            return Categories;
+            dbContext.SaveChanges();
```

Comment:
remove this - I said only when you do mutations not queries - GET is query, POST, PUT, PATCH, DELETE are mutations

Reply:
when you are retrieving data from the database it is getting/querying and you do not need to save anything, it is already in the db

Reply:
removed